NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
+	alert("This function only accepts Integers");
+	}
+	else {return (x - y);}
+}
+
+
 //Multiplication
 function multiply(x, y){
 	return (x * y);
 };
+
+function multiplication(x,y){
+	if(isNaN(multiply(x,y)) === true ){
+		alert("This function only accepts Integers");
```

Comment:
I agree you are able to put decimal points as well.  At the time I was only concern of strings. Thus the alert system wasn't explicit enough.